pydotorg/settings/base.py

```diff
@@ -31,6 +31,12 @@
     )
 }
 
+DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+"""The default primary key field type for Django models.
+
+Required during the Django 2.2 -> 4.2 migration.
+"""
```

Comment:
I'm not sure about this one, I noticed it generated a bunch of migrations. Should we set it to `django.db.models.AutoField` while we upgrade and change it after? 

Reason I'm asking is that I noticed that [the docs](https://docs.djangoproject.com/en/5.0/ref/settings/#std-setting-DEFAULT_AUTO_FIELD) mention that not all cases are covered by the migration framework:

> Unfortunately, the primary keys of existing auto-created through tables cannot currently be updated by the migrations framework.
> 
> This means that if you switch the value of DEFAULT_AUTO_FIELD and then generate migrations, the primary keys of the related models will be updated, as will the foreign keys from the through table, but the primary key of the auto-created through table will not be migrated.
> 
> In order to address this, you should add a [RunSQL](https://docs.djangoproject.com/en/5.0/ref/migration-operations/#django.db.migrations.operations.RunSQL) operation to your migrations to perform the required ALTER TABLE step. You can check the existing table name through sqlmigrate, dbshell, or with the field’s remote_field.through._meta.db_table property.

I don't know the codebase too much, so might not be an issue, but it feels like this change in primary key doesn't have to be part of the upgrade (which is quite big already).

Reply:
I agree, reverting this and setting `DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'` seems preferable (and doesn't lead to the missing migration in tastypie that is causing failing CI)